src/main/java/bio/terra/service/upgrade/Migrate.java

```diff
@@ -36,34 +37,33 @@
     private static final Logger logger = LoggerFactory.getLogger("bio.terra.service.upgrade");
     private final DataRepoJdbcConfiguration dataRepoJdbcConfiguration;
     private final MigrateConfiguration migrateConfiguration;
+    private final GoogleResourceConfiguration resourceConfiguration;
 
     @Autowired
     private Environment env;
 
     @Autowired
     public Migrate(DataRepoJdbcConfiguration dataRepoJdbcConfiguration,
-                   MigrateConfiguration migrateConfiguration) {
+                   MigrateConfiguration migrateConfiguration,
+                   GoogleResourceConfiguration resourceConfiguration) {
         this.dataRepoJdbcConfiguration = dataRepoJdbcConfiguration;
         this.migrateConfiguration = migrateConfiguration;
+        this.resourceConfiguration = resourceConfiguration;
     }
 
     /**
      * Run liquibase migration
      *
      */
-    public void migrateDatabase() {
+    public void migrateDatabase(boolean dropAllOnStart) {
         String changesetFile = dataRepoJdbcConfiguration.getChangesetFile();
         try (Connection connection = dataRepoJdbcConfiguration.getDataSource().getConnection()) {
             Liquibase liquibase = new Liquibase(changesetFile,
```

Comment:
If Liquibase was a service then you could mock it and test at the `migrateDatabase()` API level.